product-service/serverless.ts

```diff
@@ -63,6 +130,39 @@ const serverlessConfiguration: Serverless = {
             path: '/products',
             cors: true
           }
+        },
+        {
+          sqs: {
+            batchSize: 1,
+            arn: {
+              'Fn::GetAtt': [ 'catalogItemsQueue', 'Arn' ]
+            }
+          }
+        }
+      ]
+    },
+    catalogBatchProcess: {
```

Comment:
This is a name suggested by the task description as a name of the handler that processes SQS events. Apparently, this is an HTTP handler. Why?

Reply:
Updated